prime-angular-frontend/src/app/modules/adjudication/shared/components/health-authority-table/health-authority-table.component.html

```diff
@@ -1,121 +1,157 @@
-                  mat-icon-button
-                  title="View Pending Authorized Users"
-                  (click)="onRoute([AdjudicationRoutes.HEALTH_AUTHORITIES, row.id, AdjudicationRoutes.HEALTH_AUTH_AUTHORIZED_USERS])">
-            <mat-icon class="red-icon">admin_panel_settings</mat-icon>
-          </button>
-        </td>
-      </ng-container>
-
-      <ng-container matColumnDef="orgName">
-        <th mat-header-cell
-            *matHeaderCellDef
-            scope="col"> Organization Name
-        </th>
-        <td mat-cell
-            *matCellDef="let row;"> {{ row.name | default }} </td>
-      </ng-container>
-
-      <ng-container matColumnDef="siteName">
-        <th mat-header-cell
-            *matHeaderCellDef
-            scope="col"> Site Name
-        </th>
-        <td mat-cell
-            *matCellDef="let row;"> {{ null | default }} </td>
-      </ng-container>
-
-      <ng-container matColumnDef="submissionDate">
-        <th mat-header-cell
-            *matHeaderCellDef
-            scope="col"> Submission Date
-        </th>
-        <td mat-cell
-            *matCellDef="let row;"> {{ null | default }} </td>
-      </ng-container>
-
-      <ng-container matColumnDef="assignedTo">
-        <th mat-header-cell
-            *matHeaderCellDef
-            scope="col"> Assigned To
-        </th>
-        <td mat-cell
-            *matCellDef="let row;"> {{ null | default }} </td>
-      </ng-container>
-
-      <ng-container matColumnDef="state">
-        <th mat-header-cell
-            *matHeaderCellDef
-            scope="col"> State
-        </th>
-        <td mat-cell
-            *matCellDef="let row;"> {{ null | default }} </td>
-      </ng-container>
-
-      <ng-container matColumnDef="siteId">
-        <th mat-header-cell
-            *matHeaderCellDef
-            scope="col"> Site ID
-        </th>
-        <td mat-cell
-            *matCellDef="let row;"> {{ null | default }} </td>
-      </ng-container>
-
-      <ng-container matColumnDef="remoteUsers">
-        <th mat-header-cell
-            *matHeaderCellDef
-            scope="col"> Remote Users
-        </th>
-        <td mat-cell
-            *matCellDef="let row;"> {{ null | default }} </td>
-      </ng-container>
-
-      <ng-container matColumnDef="actions">
-        <th mat-header-cell
-            *matHeaderCellDef
-            scope="col">
-          &nbsp;
-        </th>
-        <td mat-cell
-            *matCellDef="let row;"
-            class="text-right">
-
-          <ng-container [ngTemplateOutlet]="menu"
-                        [ngTemplateOutletContext]="{ row: row }"></ng-container>
-
-        </td>
-      </ng-container>
-
-      <tr mat-header-row
-          *matHeaderRowDef="columns; sticky: true"></tr>
-      <tr mat-row
-          *matRowDef="let row; columns: columns;"></tr>
-    </table>
-
-    <p *ngIf="!dataSource?.data?.length"
+    <mat-accordion multi="false">
```

Comment:
Is there a way to have the name of the Health Authority within the accordion more similar to how the mockup is, rather than above it? And to make it less raised?

Reply:
Also there is a lot of space above the table when the accordion is expanded.

Reply:
> Also there is a lot of space above the table when the accordion is expanded.

feels like need some css tuneups. empty spaces when not expanded too comparing to mockup

![image](https://user-images.githubusercontent.com/58282608/128251410-c96b2485-5b67-4867-ad08-d25b88a72263.png)

Reply:
How does it look now @AnaisHebert, @johnnyl6 ?